### PR TITLE
pdfgrep: add page

### PR DESCRIPTION
**Pull request by juanjoLenero**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines 

This utility is also available in OSX via homebrew, but I'm not sure how that's handled in the repo, is the page duplicated under the OSX folder?

### CODE REVIEW

**Comment by agnivade** [comment]
Yes, you need to have another copy of the file inside the osx folder.

File: pages/linux/pdfgrep.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Find the first 3 lines that begin with foo case insensitive:
+
+`pdfgrep --max-count {{3}} --ignore-case {{'^foo'}} {{file.pdf}}`
+
+- Find pattern in files with a .pdf extension in the current directory recusively:
+
+`pdfgrep --recursive {{pattern}}`
+
+- Find pattern recursively on files that match a specific glob:
```
Comment (agnivade) [inline pages/linux/pdfgrep.md]: We should also mention here that it is in the current directory.

File: pages/linux/pdfgrep.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Find lines that match pattern in a PDF:
+
+`pdfgrep {{pattern}} {{file.pdf}}`
+
+- Include file name and page number for each matched line:
+
+`pdfgrep --with-filename --page-number {{pattern}} {{file.pdf}}`
+
+- Find the first 3 lines that begin with foo case insensitive:
```
Comment (agnivade) [inline pages/linux/pdfgrep.md]: the phrasing "foo case insensitive" sounds a bit odd. How about `Do a case insensitive search that begin with "foo" and return the first 3 lines` ?

**Comment by sbrl** [comment]
Thanks, @juanjoLenero :smiley_cat:

**Pull request merged**